elements/lisk-p2p/src/peer/inbound.ts

```diff
@@ -46,12 +56,25 @@ export class InboundPeer extends Peer {
 			this.emit(EVENT_INBOUND_SOCKET_ERROR, error);
 		};
 		this._handleInboundSocketClose = (code, reason) => {
+			if (this._pingTimeoutId) {
+				clearTimeout(this._pingTimeoutId);
+			}
 			this.emit(EVENT_CLOSE_INBOUND, {
 				peerInfo,
 				code,
 				reason,
 			});
 		};
+		this._pingStart = Date.now();
+		this._sendPing = () => {
+			clearTimeout(this._pingTimeoutId);
```

Comment:
The `clearTimeout(this._pingTimeoutId)` line is not needed here because the `_sendPing` function only executes when the timeout has been triggered; when the last timeout triggers, it is automatically cleared.

We only need to `clearTimeout` on socket disconnect.